src/pages/Dashboard.tsx

```diff
@@ -1,49 +1,35 @@
 import React from "react";
-import styled, { keyframes } from "styled-components";
-
-import logo from "assets/logo.svg";
-
-const logoSpin = keyframes`
-from {
-  transform: rotate(0deg);
-}
-to {
-  transform: rotate(360deg);
-}
+import styled from "styled-components";
+import { useRedux } from "hooks/useRedux";
+import { TransactionHistory } from "components/TransactionHistory";
 
+const El = styled.div`
+  padding-bottom: 10px;
 `;
 
-const El = styled.div`
-  text-align: center;
+const TempButtonEl = styled.button`
+  margin-bottom: 20px;
 `;
 
-const LogoEl = styled.img`
-  height: 40vmin;
-  pointer-events: none;
+export const Dashboard = () => {
+  const { account } = useRedux(["account"]);
 
-  @media (prefers-reduced-motion: no-preference) {
-    animation: ${logoSpin} infinite 20s linear;
+  let nativeBalance = 0;
+  if (account.data) {
+    nativeBalance = account.data.balances.native.total.toFixed();
```

Comment:
We might need to use `BigNumber` for the total here.

Reply:
The value comes in as BigNumber. I'll just leave as .toString() for display purposes for now. It includes the full decimals that the api returns.